src/RolloutScorer/RolloutScorer/RolloutScorer.cs

```diff
@@ -184,7 +193,8 @@ public async Task<(int numHotfixes, int numRollbacks)> CalculateNumHotfixesAndRo
 
         public bool DetermineFailure()
         {
-            return BuildBreakdowns.Count == 0 || BuildBreakdowns.Last().BuildSummary.Result != "succeeded";
+            string lastBuildResult = BuildBreakdowns.Last().BuildSummary.Result;
+            return BuildBreakdowns.Count == 0 || (lastBuildResult != "succeeded" && lastBuildResult != "partiallySucceeded");
```

Comment:
You can remove the Count check here, since the Last() from above will crash in that case.

Or maybe you didn't intend the Last() to crash... in which case you should probably split this into 2 ifs. One that says `if (BuildBreakdowns.Count == 0) return true`, and then the Last() + check later. (I'd probably use a switch statement for that guy too, so it's an easy to read list of "here are things that are failure")

Reply:
Okay I think the second one is what I'll do; that's what I was going for. Thanks Chad.